run

```diff
@@ -31,10 +31,10 @@ function manage {
 
   # We need to collectstatic before we run our tests.
   if [ "${1-''}" == "test" ]; then
-    cmd python manage.py collectstatic --no-input
+    cmd python src/manage.py collectstatic --no-input
   fi
 
-  cmd python manage.py "${@}"
+  cmd python src/manage.py "${@}"
```

Comment:
Good catch. I think this stopped working when the Docker entrypoint changed (the entrypoint was `/appsrc/`)

Would it be ok if you opened a PR just with this change? (can also include the `.gitignore`). It's more to separate the changes to the application from the changes of the setup/configuration.

Reply:
Done https://github.com/gnosis/safe-config-service/pull/170